cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -46,6 +46,52 @@ type NodeGroupAutoscalingOptions struct {
 	ScaleDownUnreadyTime time.Duration
 }
 
+const (
+	// MaxAllocatableDifferenceRatio describes how Node.Status.Allocatable can differ between
+	// groups in the same NodeGroupSet
+	MaxAllocatableDifferenceRatio = 0.05
```

Comment:
Can you rename this to `DefaultMaxAllocatableDifferenceRatio`? Similarly for the two consts below.

Reply:
works for me.

Reply:
Modified to be Default... ratio